src/dxvk/dxvk_buffer.h

```diff
@@ -182,6 +182,14 @@ namespace dxvk {
     const DxvkBufferCreateInfo& info() const {
       return m_info;
     }
+
+    /**
+     * \brief Sharing mode info
+     * \returns Sharing mode info
+     */
+    const DxvkSharingModeInfo& sharingModeInfo() const {
+      return m_sharingMode;
+    }
```

Comment:
This should be retrieved from the `DxvkDevice` instead, the only reason the buffer even stores it locally is to avoid pointer chasing when discarding buffers (which is an extremely hot code path in D3D11).

Reply:
Gotcha